projects/10_cube_map/main.cpp

```diff
@@ -131,7 +135,7 @@ void ProjApp::Setup()
         createInfo.posZ                         = PPX_ENCODE_CUBE_FACE(2, grfx_util::CUBE_FACE_OP_NONE, grfx_util::CUBE_FACE_OP_NONE);
         createInfo.negZ                         = PPX_ENCODE_CUBE_FACE(4, grfx_util::CUBE_FACE_OP_NONE, grfx_util::CUBE_FACE_OP_NONE);
 
-        PPX_CHECKED_CALL(grfx_util::CreateCubeMapFromFile(GetDevice()->GetGraphicsQueue(), GetAssetPath("basic/textures/cube_map_debug.jpg"), &createInfo, &mCubeMapImage));
+        PPX_CHECKED_CALL(grfx_util::CreateCubeMapFromFile(GetDevice()->GetGraphicsQueue(), GetAssetPath("PolyHaven/textures/hilly_terrain.png"), &createInfo, &mCubeMapImage));
```

Comment:
nit: `polyhaven`/`poly_haven`/`polyhaven.com` ? (the project folders are usually snake_case)

Reply:
I kept their name according to how they cased their lettering - just in case legal threw a fit. But I can change it.